tests/prague/eip7702_set_code_tx/test_set_code_txs.py

```diff
@@ -184,8 +193,14 @@ def test_set_code_to_sstore(
         pre=pre,
         tx=tx,
         post={
-            set_code_to_address: Account(storage={k: 0 for k in storage}),
-            auth_signer: Account(nonce=1, code=b"", storage=storage if succeeds else {}),
+            set_code_to_address: Account(
+                storage={k: 0 for k in storage},
+            ),
+            auth_signer: Account(
+                nonce=2 if self_sponsored else 1,
```

Comment:
This does not account for the out-of-gas or revert. If it reverts or OOGs then it should be 1.